### PR TITLE
Change Archas' name and peapod evs

### PR DESCRIPTION
**Pull request by Sigmatecture**

### CODE REVIEW

**Comment by xInstruct** [comment]
:blobglare:

File: data/mods/ssb/random-teams.ts [hunk 1]
```diff
@@ -520,7 +520,7 @@ export const ssbSets: SSBSets = {
 		species: 'Dragapult', ability: 'Stealth Black', item: 'Leftovers', gender: 'M',
 		moves: ['Hex', 'Dragon Darts', 'Work Up'],
 		signatureMove: 'Submartingale',
-		evs: {hp: 96, spa: 252, spd: 160}, nature: 'Mild',
+		evs: {hp: 216, spa: 252, spe: 40}, nature: 'Mild',
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts:523]: :blobglare:

Comment (ghost) [reply data/mods/ssb/random-teams.ts:523]: sure fine but im not gonna like it >:[

Comment (Sigmatecture) [reply data/mods/ssb/random-teams.ts:523]: Thank ye, O' benevolent Instruct 😄

**Pull request merged**